BetterPartyFinder/Configuration.cs

```diff
@@ -15,18 +15,17 @@ public class Configuration : IPluginConfiguration {
 
         public bool ShowWhenPfOpen { get; set; }
         public WindowSide WindowSide { get; set; } = WindowSide.Left;
-        public bool ShowDescriptionOnJoin { get; set; } = true;
```

Comment:
do you know if this will:

- opportunistically remove this value from config whenever config is loaded?
- remove this value from config the next time config is changed?
- leave the value in config indefinitely?
- something else?